digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -149,10 +149,19 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     val sourceIds = data.sourceIds
     val targetIds = data.targetIds
     val roadAddresses = roadAddressData.map{ ra =>
+      val pointsToCalibration = ra.calibrationPoints.size match {
+        case 0 => (None, None)
+        case _ => ra.calibrationPoints.head.value match {
```

Comment:
This does not seem to handle a situation where a link has both a starting calibration point and an ending one.